README.md

```diff
@@ -82,10 +82,9 @@ stay on top of the latest developments.
   Group</a> is our place for detailed questions and discussions.</td>
 </tr>
 <tr>
-  <td>IRC</td>
-  <td>Talk with the Citus development team in realtime on the <a
-  href="https://webchat.freenode.net/?channels=citus">#citus</a> Freenode
-  channel.</td>
+  <td>Slack</td>
+  <td><a href="https://slack.citusdata.com">Join</a> the Citus
```

Comment:
"Join the Citus Development room" seems a bit unclear? It seems like it would be a forum like pgsql-hackers where development/design talk goes on.

Could we stick with the previous? ("Talk with Citus development team"), or just "Chat with us in our community Slack channel", or maybe just "Join us on Slack".